src/main/java/com/wolox/training/services/OpenLibraryService.java

```diff
@@ -0,0 +1,50 @@
+package com.wolox.training.services;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility;
+import com.fasterxml.jackson.annotation.PropertyAccessor;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.wolox.training.dtos.BookInfoDTO;
+import com.wolox.training.exceptions.MyJsonProcessingException;
+import java.util.HashMap;
+import java.util.Optional;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+public class OpenLibraryService {
+
+    private static final String BASE_URL = "https://openlibrary.org/api/books?bibkeys=%s&format=json&jscmd=data";
```

Comment:
Look great, let's do something extra... read this value from the application properties. It's something very used, you can read the value using @Value("${path.to.value}") 😁